src/browser/browser.js

```diff
@@ -94,7 +94,33 @@ define((require, exports, module) => {
   const onSelectNext = throttle(edit(selectNext), 200)
   const onSelectPrevious = throttle(edit(selectPrevious), 200);
 
-  const switchToTab = index => items => maybeActivateIndex(items, index);
+  const switchTab = (items, to) =>
+    to ? activate(select(items, to)) : items;
+
+  switchTab.toIndex = index => items => switchTab(items, items.get(index));
+  switchTab.toLast = items => switchTab(items, items.last());
+
+
+  let onTabSwitch;
+  {
+    const os = navigator.platform.startsWith('Win') ? 'windows' :
+               navigator.platform.startsWith('Mac') ? 'osx' :
+               navigator.platform.startsWith('Linux') ? 'linux' :
+               '';
```

Comment:
we already do that in src/browser/keyboard.js and src/browser/actions.js. Looks like we should move it to a common place.